devito/finite_differences/coefficients.py

```diff
@@ -83,6 +84,27 @@ def dimension(self):
         """The dimension to which the coefficients will be applied."""
         return self._dimension
 
+    @property
+    def offset_dimension(self):
+        """
+        The dimension to which the coefficients will be applied plus the offset
+        in that dimension if the function is staggered.
+        """
+        if self._function.staggered is None:
```

Comment:
you should be able to get this dimension directly from `self._function.indices_ref[self._dimension]`

Reply:
Thanks, noted, will update now